internal/tiltfile/tiltfile_state.go

```diff
@@ -633,6 +661,37 @@ func (s *tiltfileState) assembleK8sV2() error {
 	return nil
 }
 
+func selectorFromString(s string) (k8sObjectSelector, error) {
```

Comment:
might be helpful to note the schema in a comment so ppl reading this code don't have to re-derive it